pipelines/managed/fbc-release/README.md

```diff
@@ -20,6 +20,10 @@ Tekton release pipeline to interact with FBC Pipeline
 | taskGitUrl                      | The url to the git repo where the release-service-catalog tasks to be used are stored                                                      | Yes      | https://github.com/konflux-ci/release-service-catalog.git |
 | taskGitRevision                 | The revision in the taskGitUrl repo to be used                                                                                             | No       | -                                                         |
 
+
+## Changes in 4.6.1
+* Pass buildResultsFile to publish-index-image task
```

Comment:
Looks like it's called internalRequestResultsFile, not buildResultsFile

Reply:
And it seems you're adding it to two tasks: publish-index-image and extract-index-image